Running on host node006
Time is Sun Mar 31 22:40:55 -03 2019
Directory is /home/david/git-repos/brain
Slurm job ID is 40266
This job runs on the following machines:
node006
Running /home/david/venv366tfcpu/bin/python /home/david/git-repos/brain/main3.py
Using TensorFlow backend.
TF version 1.13.1
TF version 1.13.1
Traceback (most recent call last):
  File "/home/david/git-repos/brain/main3.py", line 224, in <module>
    main()
  File "/home/david/git-repos/brain/main3.py", line 66, in main
    Retrieval.autoExecReg()
  File "/home/david/git-repos/brain/src/training.py", line 204, in autoExecReg
    df_orig = pd.read_csv(os.path.join(self.path, self.file), sep=',', decimal='.')
  File "/home/david/venv366tfcpu/lib/python3.6/site-packages/pandas/io/parsers.py", line 702, in parser_f
    return _read(filepath_or_buffer, kwds)
  File "/home/david/venv366tfcpu/lib/python3.6/site-packages/pandas/io/parsers.py", line 429, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/home/david/venv366tfcpu/lib/python3.6/site-packages/pandas/io/parsers.py", line 895, in __init__
    self._make_engine(self.engine)
  File "/home/david/venv366tfcpu/lib/python3.6/site-packages/pandas/io/parsers.py", line 1122, in _make_engine
    self._engine = CParserWrapper(self.f, **self.options)
  File "/home/david/venv366tfcpu/lib/python3.6/site-packages/pandas/io/parsers.py", line 1853, in __init__
    self._reader = parsers.TextReader(src, **kwds)
  File "pandas/_libs/parsers.pyx", line 387, in pandas._libs.parsers.TextReader.__cinit__
  File "pandas/_libs/parsers.pyx", line 705, in pandas._libs.parsers.TextReader._setup_parser_source
FileNotFoundError: [Errno 2] File b'/home/david/DATA/yearly_br_underc1_hot_0956.csv' does not exist: b'/home/david/DATA/yearly_br_underc1_hot_0956.csv'

real	0m6.023s
user	0m4.628s
sys	0m2.414s
